SPECS/icu/icu.signatures.json

```
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "icu-68.2.0.9.tar.gz": "f5ad6a22da531eff169fc28c759ae4604b7fd7550b2caf63a227ef706eb39f94"
+    "icu-72.1.0.3.tar.gz": "90d4fd5b23878215f6ee77e0c1286338d39a81390c769ac578dfe7235d4f298d"
```

nit: Looks like some extra whitespace snuck in, probably via the save-on-format (we use odd indenting for our signature files).
```suggestion
  "icu-72.1.0.3.tar.gz": "90d4fd5b23878215f6ee77e0c1286338d39a81390c769ac578dfe7235d4f298d"
```